src/client/chowly-client.api.ts

```diff
@@ -0,0 +1,53 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
+import { plainToClass } from 'class-transformer';
+import { IConfiguration } from '../interface';
+import { Sections } from '../model/chowly.model';
+import { sleep } from '../utils';
+
+export class ChowlyClient {
```

Comment:
Maybe it makes sense to rename class as `ApiClient` as it already in **chowly** package/namespace?

Reply:
btw inside **square-connect-plus** we have SquareClient, i think we need one-two more opinions about that

Reply:
I was looking at the original `square-connect`

```
var SquareConnect = require('square-connect');
var defaultClient = SquareConnect.ApiClient.instance;
```

and looks like they use `ApiClient`

Reply:
ok, i see, but they have `SquareConnect` so user know that it comes from Square, anyway lets wait for one more review and if we get the most votes on `ApiClient` i will change it ;)

Reply:
I am for `ChowlyApiClient`